src/main/java/core/basesyntax/dataservices/DataProcess.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.dataservices;
+
+import core.basesyntax.FruitTransaction;
+import java.util.List;
+import java.util.Map;
+
+public interface DataProcess {
+    Map<String, Integer> process(List<FruitTransaction> transactions);
```

Comment:
The method 'process' could be made more generic by not tying it to a specific 'FruitTransaction' type. Consider using generics to allow for processing different types of transactions.